Dockerfile

```diff
@@ -1,19 +1,19 @@
 # Mozilla Load-Tester
-FROM stackbrew/debian:testing
+FROM python:3.5-slim
+WORKDIR /home/loads/antenna-loadtests
 
-RUN \
-    apt-get update; \
-    apt-get install -y python3-pip python3-venv git build-essential make; \
-    apt-get install -y python3-dev libssl-dev libffi-dev; \
-    git clone -b dev https://github.com/rpappalax/antenna-loadtests.git /home/loads; \
-    cd /home/loads; \
-    pip3 install virtualenv; \
-    make build -e PYTHON=python3.5; \
-    apt-get remove -y -qq git build-essential make python3-pip python3-venv libssl-dev libffi-dev; \
-    apt-get autoremove -y -qq; \
-    apt-get clean -y
+RUN apt-get update
```

Comment:
@m8ttyB i know you're aware of this already, but just a quick reminder that we have individual RUN statements for ease of development, but that we'll need to chain these back to reduce the image size.